commands/command_lock.go

```diff
@@ -72,15 +72,18 @@ func lockPath(file string) (string, error) {
 
 	abs := filepath.Join(wd, file)
 	path := strings.TrimPrefix(abs, repo)
+	if len(path) > 1 {
```

Comment:
Is this to trim off the leading `/`? I think this should be fine to change since we don't have any production implementations in the wild right now, but maybe a clarifying comment, or something like:

```go
path = strings.TrimPefix("/", path)
```

would be good.